lib/features/forgot_password/forgot_password_page.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:flutter/material.dart';
+import 'package:easy_localization/easy_localization.dart';
+
```

Comment:
Add class documentation

Reply:
example? i don't see anything in other classes of this project.

Reply:
in `add_new_cbj_devices.dart` class `AddNewCBJDevices` you can see one line above it `/// hotspot of the app device.`